apiary.apib

```diff
@@ -2632,6 +2641,8 @@ Update a milestone.
 
 ## Time Tracking [/timeTracking]
 
+Time tracking gives you insight into the amount of work done and the work type. You are able to follow-up on the budgets of your projects and eventually you are able to invoice the billable time.
```

Comment:
`and the work type` might be better as `and its work type`

Reply:
> There is a space between follow and up under one condition—you are using follow up as a verb. To follow up means to gather further information or to reinforce or evaluate a previous action.

Reply:
I asked Niels to improve this text.